### PR TITLE
add sort option

### PR DESCRIPTION
**Pull request by Daley**
app.use('/', serveIndex("dir", {'icons': true,sort:serveIndex.fileSortWithMTime}));

### CODE REVIEW

**Comment by dougwilson** [comment]
Hi @Daley, thanks for your pull request! There is another one before your PR, can you explain the differences between that one and this one?

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "serve-index",
   "description": "Serve directory listings",
-  "version": "1.7.3",
+  "version": "1.7.4",
```
Comment (dougwilson) [inline package.json]: Please undo this version bump.

**Comment by mrv777** [comment]
https://github.com/expressjs/serve-index/pull/41/ seems to sort by a function that is passed.  This one just adds the sort by modified date.  This one is simpler and the other is more versatile.
I prefer this one for simplicity and can code other sort functions as suggested by the author instead of making the user provide it.

It's up to you, but I think this function is very much needed

**Comment by baihe** [comment]
Will this be merged ?
Thanks

**Comment by roy7** [comment]
It'd be awesome if you could merge in one of the two sort solutions. :)

**Comment by dougwilson** [comment]
I at least need the PR updated with the change request I made. Since I requested the change a year ago and the PR still hasn't been updated, I'm just going to close it. Someone can always open a new PR if they are interested in picking it up and making any changes from review feedback.

**Comment by roy7** [comment]
Ah ok, bummer @mrv777 didn't make that correction for you.

Since I only needed sorting myself for a specific one-off project, I just manually changed fileSort to the code he used.

**Pull request closed without merging**